tests/plugins/editorplaceholder/editorplaceholder.js

```diff
@@ -205,5 +205,44 @@
 		} );
 	};
 
+	// (#4253)
+	tests[ 'test placeholder loads correctly in full-page editor with bbcode plugin'] = function() {
+
+		var errorLogStub = sinon.stub( console, 'error' );
+
+		var onErrorStub = function() {
+			resume( function() {
+				assert.fail( 'There should be no RunTime errors!' );
+			} );
+		};
+
+		window.addEventListener( 'error', onErrorStub );
+
+		var instanceReadyCallback = function( e ) {
+			resume( function() {
+				assert.isFalse( errorLogStub.called, 'There should be no errors during initialization' );
+			} );
+		};
+
+		bender.editorBot.create( {
+			name: 'bbcodeFailTest',
+			config: {
+				extraPlugins: 'bbcode',
+				editorplaceholder: 'any',
+				fullPage: true,
+				on: {
+					instanceReady: instanceReadyCallback
+				}
+			}
+		}, function( bot ) {
+
+		} );
+
+		errorLogStub.restore();
+		window.removeEventListener( 'error', onErrorStub );
```

Comment:
Well, if the test fails, this lines will not be executed. You can move stubs/listener removals to `tearDown` method like:

```javascript
if ( errorLogStub ) {
	errorLogStub.restore();
}
```